src/main/java/com/redhat/cloud/policies/app/health/ProcSelfStatusExporter.java

```diff
@@ -47,15 +47,35 @@
 
     private boolean hasWarned = false;
 
-    long vmHwm;
-    long vmRss;
-    long rssAnon;
-    long rssFile;
-    long vmStk;
-    long vmLib;
-    long vmData;
-    long vmSize;
-    int threads;
+    @Inject
+    MeterRegistry registry;
+
+    // @Gauge(name = "status.vmHwm", absolute = true, unit = MetricUnits.KILOBYTES, tags = "type=proc")
+    Long vmHwm = registry.gauge("status.vmHwm", 0L);
```

Comment:
This does't need to be on a `@PostConstruct` method?

Reply:
Just tested it. Looks like it's not necessary to have it in a postconstruct method